--- ../distributions/RHEL6/cvsw_net.c	2014-09-10 08:56:20.030003276 +0900
+++ cvsw_test_main.c	2014-09-10 10:30:39.547006625 +0900
@@ -31,10 +31,11 @@
  * CVSW extension (BEGIN)
  ***************************************************************************/
 
-#include "../../cvsw_net.h"
-#include "../../cvsw_ctl.h"
-#include "../../cvsw_table.h"
-#include "../../cvsw_data.h"
+#include "../cvsw_net.h"
+#include "../cvsw_ctl.h"
+#include "../cvsw_table.h"
+#include "../cvsw_data.h"
+#include "cvsw_test.h"
 
 #define VIRTIO_NET_HDR_GSO_UDP_TUNNEL 0x20
 
@@ -1166,6 +1167,12 @@
 	}
 
 	pr_debug("cvsw: registered device %s\n", dev->name);
+
+	if (! cvsw_test_start(dev)) {
+		err = -ENOMEM;
+		goto unregister;
+	}
+
 	return 0;
 
 unregister:
