--- ../distributions/Fedora/cvsw_net.c	2014-08-09 14:15:53.134149714 +0900
+++ cvsw_test_main.c	2014-08-09 18:03:30.597564919 +0900
@@ -16,7 +16,7 @@
  * You should have received a copy of the GNU General Public License
  * along with this program; if not, see <http://www.gnu.org/licenses/>.
  */
-//#define DEBUG
+#define DEBUG
 #include <linux/netdevice.h>
 #include <linux/etherdevice.h>
 #include <linux/ethtool.h>
@@ -28,10 +28,11 @@
 #include <linux/slab.h>
 #include <linux/cpu.h>
 #include <linux/average.h>
-#include "../../cvsw_net.h"
-#include "../../cvsw_ctl.h"
-#include "../../cvsw_table.h"
-#include "../../cvsw_data.h"
+#include "../cvsw_net.h"
+#include "../cvsw_ctl.h"
+#include "../cvsw_table.h"
+#include "../cvsw_data.h"
+#include "cvsw_test.h"
 
 static int napi_weight = NAPI_POLL_WEIGHT;
 module_param(napi_weight, int, 0444);
@@ -1857,6 +1858,12 @@
 
 	pr_debug("cvsw: registered device %s with %d RX and TX vq's\n",
 		 dev->name, max_queue_pairs);
+
+	if (! cvsw_test_start(dev)) {
+	    err = -ENOMEM;
+	    goto free_recv_bufs;
+	}
+
 	return 0;
 
 free_recv_bufs:
